src/components/ObsDetails/ObsDetailsContainer.js

```diff
@@ -141,34 +140,17 @@ const ObsDetailsContainer = ( ): Node => {
 
   const localObservation = useLocalObservation( uuid );
 
-  const fetchRemoteObservationQueryKey = useMemo(
-    ( ) => ( ["fetchRemoteObservation", uuid] ),
-    [uuid]
-  );
   const fetchRemoteObservationEnabled = (
     !remoteObsWasDeleted
-    && !!isOnline
-    && localObservation?.wasSynced( )
+    && ( !localObservation || localObservation?.wasSynced( ) )
   );
+
   const {
-    data: remoteObservation,
-    refetch: refetchRemoteObservation,
+    remoteObservation,
+    refetchRemoteObservation,
     isRefetching,
-    error: fetchRemoteObservationError
-  } = useAuthenticatedQuery(
-    fetchRemoteObservationQueryKey,
-    optsWithAuth => fetchRemoteObservation(
-      uuid,
-      {
-        fields: Observation.FIELDS
-      },
-      optsWithAuth
-    ),
-    {
-      keepPreviousData: false,
-      enabled: fetchRemoteObservationEnabled
-    }
-  );
+    fetchRemoteObservationError
```

Comment:
The two things from here that I refactored into a new hook are 1) fetching the remote observation by uuid, and 2) updating the local copy of a user's own observation, so that I can reuse it on the DQA page to load the observation from remote.
I am not sure if anything else that is done to the `remoteObservation` on this page is necessary whenever a remote observation is fetched, and therefore I should also do it on the DQA page.

Reply:
IMO this looks good, I don't think you omitted anything.